docs/plugins/argocdapp_plugin.md

```diff
@@ -11,13 +11,13 @@ This plugin creates an ArgoCD Application.
 
 ```yaml
 tools:
-- name: argocdapp
+- name: argocdapp-helloworld
```

Comment:
How about changing it to `helloworld`?
because the key in the state will be `argocdapp-helloworld-argocdapp`, it's redundant.